GW2EIEvtcParser/EncounterLogic/Raids/W7/PeerlessQadim.cs

```diff
@@ -335,25 +386,195 @@ internal override void ComputePlayerCombatReplayActors(AbstractPlayer p, ParsedE
             var magmaDrop = p.GetBuffStatus(log, MagmaDrop, log.FightData.FightStart, log.FightData.FightEnd).Where(x => x.Value > 0).ToList();
             int magmaRadius = 420;
             int magmaOffset = 4000;
+            int magmaDuration = 600000;
```

Comment:
Where is that value coming from?

Reply:
That does not answer the question ^^
I couldn't find anything on the subject in the wiki, did you get that from arc effect events? Experimentation?

Reply:
yeah it's the effect duration
![6154](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/baa710e7-74a8-4df9-b7f9-dcae7cdd786f)